File: Lecture02/activity_main/app/src/main/java/com/example/neera/activity_main/MainActivity.java
package com.example.neera.activity_main;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.text.Editable;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import java.security.acl.LastOwnerException;

public class MainActivity extends AppCompatActivity {

    EditText etFirstName,etLastName;
    Button btnAdd;
    TextView tvFullName;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        etFirstName = (EditText) findViewById(R.id.etFirstName);
        etLastName = (EditText) findViewById(R.id.etLastName);
        btnAdd = (Button) findViewById(R.id.btAdd);
        tvFullName = (TextView) findViewById(R.id.tvFullName);

        btnAdd.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View v){
                int firstName = Integer.parseInt(etFirstName.getText().toString());
                int lastname = Integer.parseInt(etLastName.getText().toString());

                int fullName = firstName + lastname;

                tvFullName.setText(Integer.toString(fullName));
            }
        });


    }
}

File: Android Class Applications/Lecture14/SampleAndroidTestCoverage/app/src/main/java/com/codingblocks/testingcoverage/MainActivity.kt
package com.codingblocks.testingcoverage

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnCalcFare.setOnClickListener({
            tvResult.text = calcFare(
                    etKm.text.toString().toFloat(),
                    etMin.text.toString().toInt()
            ).toString()
        })
    }

    companion object {
        @JvmStatic fun calcFare(km: Float, min: Int) : Float =
                50f + (if (km > 5 ) ((km-5) * 12) else 0f ) + (if (min > 15)  (min - 15) else 0 )
    }
}
